app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str,
+                 appetite: int = 3,
+                 is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str,
+                 appetite: int = 7,
+                 is_hungry: bool = True) -> None:
+        super().__init__(name, is_hungry)
+        self.appetite = appetite
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(list_of_animals: list) -> int:
+    return sum([food.feed() for food in list_of_animals])
```

Comment:
No need to convert to list
```suggestion
    return sum(food.feed() for food in list_of_animals)
```